ansible/roles/dev-desktop/files/github_app/git-credential-dev-desktop

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python
+
+from sys import argv, stdin
+import os
+
+if argv[1] != "get":
+    exit(0)
+
+config = [x.strip() for x in stdin]
+
+if config[1] != "host=github.com":
+    exit(0)
+
+path = config[2].split('=')
+path = path[1].split('/')
+
+user = path[0]
```

Comment:
oh this is the wrong user. we need to get the user from the logged in username, otherwise users can steal other user's repos. will fix this tomorrow